editor/gui/editor_scene_tabs.cpp

```diff
@@ -191,9 +191,11 @@ void EditorSceneTabs::_disable_menu_option_if(int p_option, bool p_condition) {
 }
 
 void EditorSceneTabs::update_scene_tabs() {
+	static bool menu_initialized = false;
 	tab_preview_panel->hide();
 
-	if (scene_tabs->get_tab_count() == EditorNode::get_editor_data().get_edited_scene_count()) {
+	if (menu_initialized && scene_tabs->get_tab_count() == EditorNode::get_editor_data().get_edited_scene_count()) {
+		menu_initialized = true;
```

Comment:
Shouldn't this be set after the check? Otherwise this condition can never be reached.